lib/registries/addon/components/editable-field/styles.scss

```diff
@@ -0,0 +1,20 @@
+.Editable {
+    display: flex;
+
+    :global(.btn.btn-link) {
+        padding: 0;
+        margin-left: 15px;
```

Comment:
instead of a margin, put `justify-content: space-between;` on the parent flex box, so the pencil icon is aligned to the right.